### PR TITLE
Mahi sponsors

### PR DESCRIPTION
**Pull request by mahifreya**

### CODE REVIEW

**Comment by mahifreya** [comment]
Updating sponsors to use sponsor card component, fixing typo in "sponsors" in some files, refactoring sponsors so it uses the json rather than hard coded data, making sure react palette color scheme works with each sponsor.

File: src/components/Sponsors/SponsorCard.jsx [hunk 1]
```diff
@@ -3,15 +3,16 @@ import {Palette} from 'react-palette';
 import "./Sponsors.css";
 
 function SponsorCard({ link, img, imgName}) {
-  return ( imgName === "blackstone" ?
-  <Palette src={img}>
+  console.log(img)
```
Comment (NgaiJustin) [inline src/components/Sponsors/SponsorCard.jsx]: Remember to remove the `console.log`s before pushing

File: src/components/Sponsors/Sponsors.jsx [hunk 2]
```diff
@@ -8,6 +8,8 @@ function Sponsors() {
     const getGold = json.sponsors.filter((sponsor) => sponsor.class === "gold");
     const getSilver = json.sponsors.filter((sponsor) => sponsor.class === "silver");
     const getBronze = json.sponsors.filter((sponsor) => sponsor.class === "bronze");
+    console.log(getGold)
+    console.log("hello")
```
Comment (NgaiJustin) [inline src/components/Sponsors/Sponsors.jsx]: See above

**Review by NgaiJustin — APPROVED** [review]
Yup looks good just remember to remove the logging stuff before pushing

**Pull request merged**